src/content/blog/astro-5120.mdx

```diff
@@ -0,0 +1,160 @@
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## TOML support for content loaders
+
+Astro 5.12 brings native support for TOML files in content collections. Astro's [`file()` and `glob()` content loaders](https://docs.astro.build/en/reference/content-loader-reference/#built-in-loaders) have had you covered for Markdown, JSON, and YAML files, and now you can use TOML files as well.
+
+There's no configuration needed to enable TOML support: just add `.toml` files to a collection using a built-in loader. For example, if you have a directory of TOML files like this:
+
+```toml title="src/content/spacecraft/apollo.toml"
+name = "Apollo"
+launch_date = "1969-07-16"
+crew = ["Neil Armstrong", "Buzz Aldrin", "Michael Collins"]
+```
+
+You can then create a content collection that includes the TOML files:
+
+```ts title="src/content/config.ts" "glob({ pattern: '*.toml', base: './src/content/spacecraft' })"
+import { defineCollection, z } from 'astro:content';
+import { glob } from 'astro/loaders';
+
+const spacecraft = defineCollection({
+	loader: glob({ pattern: '*.toml', base: './src/content/spacecraft' }),
+	schema: z.object({
+		name: z.string(),
+		launch_date: z.coerce().date(),
+		crew: z.array(z.string()),
+	}),
+});
+export const collections = { spacecraft };
+```
+
+You can then access the TOML content in your Astro components:
+
+```astro title="src/pages/spacecraft.astro"
+---
+import { getCollection } from 'astro:content';
+
+const spacecraft = await getCollection('spacecraft');
+---
+
+<ul>
+	{
+		spacecraft.map((item) => (
+			<li>
+				<h2>{item.name}</h2>
+				<p>Launch Date: {item.launch_date.toLocaleDateString()}</p>
+				<p>Crew: {item.crew.join(', ')}</p>
+			</li>
+		))
+	}
+</ul>
+```
+
+## New Netlify dev experience
+
+The [Netlify adapter](https://docs.astro.build/en/guides/integrations-guide/netlify/) is now powered by Netlify's very own [Vite plugin](https://docs.netlify.com/frameworks/vite/#vite-plugin) during development, bringing Netlify's entire platform and primitives to your local dev environment!
+
+### The power of Netlify on localhost
+
+Until now, using Netlify primitives with Astro during local development required running your project through the [Netlify CLI](https://docs.netlify.com/cli/get-started/). The new adapter embeds these capabilities directly into Astro's dev server instead.
+
+After upgrading to the latest version of the adapter, running `astro dev` will provide:
+
+- A **local Netlify Image CDN** for [Astro images](https://docs.astro.build/en/guides/images/)
+- A **local Blobs server** for [sessions](https://docs.astro.build/en/guides/sessions/) 
+- **[Redirects, rewrites](https://docs.netlify.com/routing/redirects/) and [headers](https://docs.netlify.com/routing/headers/#app)**  from your Netlify config
+- The **[Netlify Edge Context](https://docs.astro.build/en/guides/integrations-guide/netlify/#accessing-edge-context-from-your-site)** in on-demand pages
```

Comment:
```suggestion
- A local Netlify Image CDN for [Astro images](https://docs.astro.build/en/guides/images/)
- A local Blobs server for [sessions](https://docs.astro.build/en/guides/sessions/) 
- Redirects, rewrites](https://docs.netlify.com/routing/redirects/) and [headers](https://docs.netlify.com/routing/headers/#app)  from your Netlify config
- The [Netlify Edge Context](https://docs.astro.build/en/guides/integrations-guide/netlify/#accessing-edge-context-from-your-site) in on-demand pages
```
The bold really didn't work visually in the blog post, so I think we can just go with the list! (The non-bold "A" in front became really difficult to read in comparison, one line had bold links but the others didn't...)